main.py

```diff
@@ -148,6 +148,7 @@ def configure_confluent_kafka_consumer(event, args):
         "path.format": args.path_format,
         "locale": args.locale,
         "timezone": args.timezone,
+        "name": args.connector_name,
```

Comment:
I don't like the idea of adding a key to the config that doesn't belong. How about instead removing the "name" key from `current_config`?